modules/apps/frontend-data-set/frontend-data-set-api/src/main/java/com/liferay/frontend/data/set/constants/FDSTimeZoneBehaviors.java

```diff
@@ -0,0 +1,19 @@
+/**
+ * SPDX-FileCopyrightText: (c) 2024 Liferay, Inc. https://liferay.com
+ * SPDX-License-Identifier: LGPL-2.1-or-later OR LicenseRef-Liferay-DXP-EULA-2.0.0-2023-06
+ */
+
+package com.liferay.frontend.data.set.constants;
+
+/**
+ * @author Marko Cikos
+ */
+public class FDSTimeZoneBehaviors {
+
+	public static final String APPLY_THEME_DISPLAY_TIME_ZONE =
+		"applyThemeDisplayTimeZone";
+
+	public static final String DO_NOT_MODIFY_DATE_VALUE =
+		"doNotModifyDateValue";
```

Comment:
It might be a bit too verbose for typical naming patterns we have, but the idea to be as descriptive as possible.